clients/cobol-lsp-vscode-extension/src/services/LanguageClientService.ts

```diff
@@ -30,6 +31,8 @@ import {Middleware} from "./Middleware";
 export class LanguageClientService {
     private readonly jarPath: string;
 
+    public languangeClient: LanguageClient;
```

Comment:
Can it be private?

Reply:
You are right. We need to create the client out of the LanguageClientService and provide it via the constructor.
It's a temporal hack.